src/lr_autoreduce/reduce_REF_L.py

```
@@ -0,0 +1,213 @@
+
+
+def confirm_data_availability(sample_logs: SampleLogs) -> None:
+    """Notify the external data confirmation utility that data are available.
+
+    Raises subprocess exceptions on failure so the caller can handle/log them.
+    """
+    ipts = sample_logs["experiment_identifier"]
+    ipts_number = ipts.split("-")[1]
+    cmd = ["/SNS/software/nses/bin/confirm-data", "-s", "Yes", "BL-4B", ipts_number, "1", "Auto"]
+    try:
+        subprocess.run(cmd, check=True, timeout=30)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)